lib/demos/material/app_bar_demo.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/gallery_localizations.dart';
+
+// BEGIN appbarDemo
+
+class AppBarDemo extends StatelessWidget {
+  const AppBarDemo({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    var localization = GalleryLocalizations.of(context);
+    return Scaffold(
+      appBar: AppBar(
+        leading: IconButton(
+          icon: const Icon(Icons.menu),
```

Comment:
Please set the tooltip params for each IconButton, just like in https://github.com/flutter/gallery/blob/master/lib/demos/material/bottom_app_bar_demo.dart#L142